docs/src/test-api/class-testconfig.md

```diff
@@ -212,7 +212,15 @@ export default defineConfig({
 * since: v1.10
 - type: ?<[int]>
 
-The maximum number of test failures for the whole test suite run. After reaching this number, testing will stop and exit with an error. Setting to zero (default) disables this behavior.
+The maximum number of test failures (test retry is also a failure) for the whole test suite run. After reaching this number, testing will stop and exit with an error. Setting to zero (default) disables this behavior.
+
+Can be specified as:
+* static value (positive integer number);
+* dynamic value (positive float number less than 1) relative to the total number of tests.
```

Comment:
Perhaps the description is not very clear and I should just use `integer number` and `float number` without `static` and `dynamic`.

Reply:
For `--workers` we allow setting the value `50%`. This helps us stay away from floating point, lets do the same here.

Reply:
> For --workers we allow setting the value 50%. This helps us stay away from floating point, lets do the same here.

Ok, I didn't notice such an implementation. Fixed